install/Knative-with-Minishift.md

```diff
@@ -20,7 +20,16 @@ You can find [guides for other platforms here](README.md).
 minishift version
 ```
 
-## Configure and start minishift
+## Automatic Set Up
+
+Once you have `minishift` present on your machine and in your `PATH`,
+you can [run a script](scripts/knative-with-minishift.sh) that automates the steps on this page.
+
+## Manually Set Up
```

Comment:
```suggestion
## Manual set up
```

Reply:
@vorburger  Please also make the suggested changes to the headings/structure here in #982